contracts/bridge/Bridge.sol

```diff
@@ -156,14 +180,14 @@ contract Bridge is ITokenReceiver, INFTReceiver, Ownable {
         require(allowedTokens[msg.sender] != address(0), "Not a valid token");
         require(_amount > 0, "zero amount");
 
-        emit RequestValueTransfer(TokenKind.TOKEN, _from, _amount, msg.sender, _to, requestNonce);
+        emit RequestValueTransfer(TokenKind.ERC20, _from, _amount, msg.sender, _to, requestNonce,"");
```

Comment:
Using `RequestValueTransfer()` in common may increases unnecessary arguement for KLAY and ERC20. How about add another method?

Reply:
This event is used in sub-bridge code. We can makde RequestKLAYTransfer(), RequestERC721Transfer(), RequestERC20Transfer(). But In this case, I think they share the same request nonce. So I think it is weird.
If we decide to separate the function, we can do in another PR. :)